Patch-Source: https://github.com/troglobit/editline/commit/f53bebdbe98c4dbfc9cc959032a071415324dde2
--
From f53bebdbe98c4dbfc9cc959032a071415324dde2 Mon Sep 17 00:00:00 2001
From: Terry Greeniaus <terrygreeniaus@gmail.com>
Date: Sat, 5 Apr 2025 23:17:20 -0600
Subject: [PATCH] Fix function prototypes.

This allows it to compile on macOS 15.3.2 using the default command-line
tools (clang-1600.0.26.6).
---
 examples/fileman.c | 4 ++--
 src/editline.c     | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/examples/fileman.c b/examples/fileman.c
index ca1ec82..45e936d 100644
--- a/examples/fileman.c
+++ b/examples/fileman.c
@@ -62,8 +62,8 @@ struct cmd commands[] = {
 };
 
 /* Forward declarations. */
-char *stripwhite();
-struct cmd *find_command();
+char *stripwhite(char *string);
+struct cmd *find_command(char *name);
 
 /* ~/.fileman_history */
 char *fileman_history;
diff --git a/src/editline.c b/src/editline.c
index 5fd690b..f19d328 100644
--- a/src/editline.c
+++ b/src/editline.c
@@ -1431,7 +1431,7 @@ void rl_clear_message(void)
     /* Nothing to do atm. */
 }
 
-void rl_forced_update_display()
+void rl_forced_update_display(void)
 {
     redisplay(0);
     tty_flush();
